addons/web/static/src/views/fields/selection/selection_field.xml

```diff
@@ -6,7 +6,7 @@
             <span t-esc="string" t-att-raw-value="value" />
         </t>
         <t t-else="">
-            <select class="o_input" t-on-change="onChange">
+            <select class="o_input o_field_widget" t-on-change="onChange">
```

Comment:
Without this change, I got this:
<img width="499" alt="Screenshot 2022-09-29 at 14 48 01" src="https://user-images.githubusercontent.com/3245568/193035522-53d0935a-8ec6-4109-816c-74aa3c82c9eb.png">

The warning icon is on the next line. Do I need to change the css?

Reply:
yes, you can probably add a bootstrap class on the select (d-inline maybe?)